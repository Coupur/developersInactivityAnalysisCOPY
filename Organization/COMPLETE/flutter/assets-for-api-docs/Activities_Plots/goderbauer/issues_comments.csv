id,date,creator_login
303905417,2017-05-25 02:29:31,goderbauer
