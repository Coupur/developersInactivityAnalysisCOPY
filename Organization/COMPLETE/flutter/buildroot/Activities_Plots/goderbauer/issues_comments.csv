id,date,creator_login
475683283,2019-03-22 16:16:59,goderbauer
433984605,2018-10-29 16:42:47,goderbauer
431321119,2018-10-19 10:38:22,goderbauer
420203948,2018-09-11 09:11:29,goderbauer
419878330,2018-09-10 11:16:01,goderbauer
420058813,2018-09-10 20:58:42,goderbauer
417757611,2018-08-31 18:48:21,goderbauer
417743971,2018-08-31 17:58:18,goderbauer
405416484,2018-07-16 23:58:58,goderbauer
311819001,2017-06-28 23:22:13,goderbauer
275797304,2017-01-27 22:55:26,goderbauer
