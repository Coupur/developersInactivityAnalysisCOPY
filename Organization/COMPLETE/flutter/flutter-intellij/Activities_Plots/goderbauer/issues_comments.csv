id,date,creator_login
412601431,2018-08-13 17:37:42,goderbauer
391470668,2018-05-23 19:36:41,goderbauer
391473515,2018-05-23 19:46:54,goderbauer
368990206,2018-02-27 19:06:48,goderbauer
368990418,2018-02-27 19:07:26,goderbauer
368990739,2018-02-27 19:08:30,goderbauer
359590865,2018-01-22 22:22:33,goderbauer
359642788,2018-01-23 01:48:28,goderbauer
356436629,2018-01-09 22:34:40,goderbauer
356437553,2018-01-09 22:38:31,goderbauer
356674983,2018-01-10 17:27:22,goderbauer
323158489,2017-08-17 18:38:55,goderbauer
323172028,2017-08-17 19:33:25,goderbauer
323408509,2017-08-18 17:06:05,goderbauer
364510821,2018-02-09 18:03:09,goderbauer
300520797,2017-05-10 15:33:02,goderbauer
296800251,2017-04-24 19:39:44,goderbauer
274974580,2017-01-24 23:37:34,goderbauer
