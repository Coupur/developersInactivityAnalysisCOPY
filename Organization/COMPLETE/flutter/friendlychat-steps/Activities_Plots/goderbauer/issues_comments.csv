id,date,creator_login
314549785,2017-07-11 19:39:24,goderbauer
301637974,2017-05-16 00:03:42,goderbauer
