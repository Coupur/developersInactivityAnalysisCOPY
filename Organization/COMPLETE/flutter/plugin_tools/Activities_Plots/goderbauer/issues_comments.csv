id,date,creator_login
398129535,2018-06-18 17:18:09,goderbauer
398129663,2018-06-18 17:18:30,goderbauer
380936375,2018-04-12 20:35:56,goderbauer
363284313,2018-02-06 01:54:09,goderbauer
363285848,2018-02-06 02:02:58,goderbauer
306553233,2017-06-06 17:08:17,goderbauer
303817098,2017-05-24 18:49:55,goderbauer
