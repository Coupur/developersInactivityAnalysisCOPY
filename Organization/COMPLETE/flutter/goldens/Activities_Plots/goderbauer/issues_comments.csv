id,date,creator_login
402199969,2018-07-03 15:35:41,goderbauer
